### PR TITLE
feat: load user-defined Rego rules for workload auto-import

### PR DESCRIPTION
**Pull request by ivanstanev**
- [x] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [x] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does

The Rego rules are loaded by snyk-monitor at the start of the application, the rules are verified, and finally they are transmitted to upstream.
We want the rules to be compiled and processed client-side but any choice whether workloads should be imported or deleted happens in the upstream.
Test coverage provided in system tests to ensure the policy is being transmitted first thing on start up of the application.

### Notes for the reviewer

Not updating the docs yet since the feature is not ready to be used (needs backend support).

### More information

- [Jira ticket RUN-1404](https://snyksec.atlassian.net/browse/RUN-1404)

### CODE REVIEW

File: package.json
Comment (dkontorovskyy) [inline package.json:22]: `npm run dev` should be for running project, not waiting for brk point. 
`npm run debug` makes for sense for that 
IMHO

File: src/common/policy.ts
Comment (dkontorovskyy) [inline src/common/policy.ts:1]: Double fs import can be shortend on lines 1 and 3 :)

File: src/common/policy.ts
Comment (dkontorovskyy) [inline src/common/policy.ts:21]: you can also use async exists for non-blocking purposes?

File: src/common/policy.ts
Comment (dkontorovskyy) [inline src/common/policy.ts:28]: I would recommend not doing nested `try` as it's harder to handle errors and also taking a lot of resources from process

File: src/transmitter/index.ts
Comment (dkontorovskyy) [inline src/transmitter/index.ts:83]: no need in `else` it won't continue this path after throwing error

File: package.json
Comment (ivanstanev) [reply package.json:22]: Committed by accident while testing locally, but indeed useful as `debug`

File: src/common/policy.ts
Comment (ivanstanev) [reply src/common/policy.ts:1]: Good spot

File: src/transmitter/index.ts
Comment (jmickey) [inline src/transmitter/index.ts:80]: Do we want to wait for #664 to be merged before we merge this one, and we can use the queue? Or do we want to skip the queue for this payload?

Comment (dkontorovskyy) [reply src/transmitter/index.ts:80]: Another things, does homebase has `/api/v1/policy` route? Or this PR is actually blocked until we add `/api/v1/policy` into homebase?

File: src/common/policy.ts
Comment (jmickey) [reply src/common/policy.ts:21]: I though async exists was deprecated?

File: src/transmitter/index.ts
Comment (ivanstanev) [reply src/transmitter/index.ts:80]: Doesn't exist yet but doesn't have to be blocking since it will print an error that it can't send the policy only once on deployment (assuming we can tolerate that until the Homebase endpoint is added)

**Review by dkontorovskyy — APPROVED** [review]

**Pull request merged**
